tests/unit/latency-monitor.tcl

```diff
@@ -196,7 +196,7 @@ start_cluster 1 1 {tags {"latency-monitor cluster external:skip needs:latency"}
         # We don't assert anything since we can't be sure whether it will be counted.
         R 0 cluster saveconfig
         R 1 cluster saveconfig
-        R 1 cluster failover force
+        R 1 cluster failover takeover
```

Comment:
The reason for changing it is that in the previous test case, force does not have enough votes to finish the election.